README.md

```diff
@@ -509,6 +509,46 @@ end
 
 For more examples, check out [spec/integration/integration_spec.rb](spec/integration/integration_spec.rb).
 
+** Multiple Trackers **
+
+You can have different trackers for different classes like so.
+
+```
+class First
+  include Mongoid::Document
+  include Mongoid::History::Trackable
+
+  field :text, type: String
+  track_history on: [:text],
+                tracker_class_name: :first_history_tracker
+end
+
+class Second
+  include Mongoid::Document
+  include Mongoid::History::Trackable
+
+  field :text, type: String
+  track_history on: [:text],
+                tracker_class_name: :second_history_tracker
+end
+
+class FirstHistoryTracker
+  include Mongoid::History::Tracker
+end
+
+class SecondHistoryTracker
+  include Mongoid::History::Tracker
+end
+```
+
+Note that if you are using a tracker for an embedded object that is different
+from the parent's tracker, redos and undos will not work. You have to use the
+same tracker for these to work across embedded relationships.
+
+If you are using multiple trackers and the `tracker_class_name` parameter is
+not specified, Mongoid::History will use the default tracker configured in the
+initializer file or whatever the first tracker was loaded.
```

Comment:
Feels like this is saying that the first tracker loaded is going to be load-order dependent. That's not what's happening here, no? What's the default tracker?

Reply:
I just went by what the original readme said: "Mongoid::History will use the first loaded class to include Mongoid::History::Tracker as the default history tracker"

That's if there's no initializer. It looks to me that line 22 in tracker.rb would just set it to whatever gets loaded first.

Reply:
Got it. Lets leave it as is and maybe open an issue about that?

Reply:
Yeah maybe the default tracker shouldn't be optional but should be a required in the configuration.